projects/ama-sdk/src/lib/process-editor/bpmn.d.ts

```diff
@@ -63,4 +64,10 @@ declare namespace Bpmn {
     export interface Modeling {
         updateProperties(element: DiagramElement, properties: { [key: string]: any }): void;
     }
+
+    export interface Moddle {
+        create(tagName: string, property: { [key: string]: any }): void;
+        createAny(tagName: string, property: { [key: string]: any }): void;
```

Comment:
Do we still need this createAny? I'd rather remove it before we start to use it frequently.